### PR TITLE
Fix error when creating a new file

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: src/filebrowser/browser.ts [hunk 1]
```diff
@@ -112,10 +112,11 @@ class FileBrowserWidget extends Widget {
     this._listing = new DirListing({ manager, model, opener, renderer });
 
     model.fileChanged.connect((fbModel, args) => {
+      let oldPath = args.oldValue ? args.oldValue.path : void 0;
```
Comment (ellisonbg) [inline src/filebrowser/browser.ts]: Since I broke this I would like to understand the logic. Are you making sure the oldValue exists and has the path?

Comment (blink1073) [inline src/filebrowser/browser.ts]: Just that it exists at all, we were trying to access a property of undefined.

**Comment by afshin** [comment]
I didn't see this PR when I submitted https://github.com/jupyter/jupyterlab/pull/1065

When this PR is merged, please close 1065 as well.

**Comment by blink1073** [comment]
Superseded by #1065.

**Pull request closed without merging**